environments/test-west-region1/main.tf

```diff
@@ -0,0 +1,30 @@
+terraform {
+  required_version = ">= 1.2.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "5.50.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = var.aws_region
+}
+
+module "python_lambda" {
+  source                  = "../../lambda"
+  aws_region              = var.aws_region
+  lambda_source_file_path = "../../"
+  lambda_file_name        = "app"
+}
+
+terraform {
+  backend "s3" {
```

Comment:
You can pass backend to the same single terraform block at the top 😎

Reply:
right! was not sure if I should as not clear on best practices.I will change it.